src/main/resources/db/migration/0300/V321__AdHocPlotObservations.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE tracking.observation_types (
+    id INTEGER PRIMARY KEY,
+    name TEXT NOT NULL UNIQUE
+);
+
+INSERT INTO tracking.observation_types (id, name)
+VALUES (1, 'Monitoring'),
+       (2, 'Biomass Measurements')
+ON CONFLICT (id) DO UPDATE SET name = excluded.name;
+
+ALTER TABLE tracking.observations ADD COLUMN is_ad_hoc BOOLEAN;
+
+UPDATE tracking.observations
```

Comment:
Would it make more sense to put `is_ad_hoc` in the monitoring plots table? 

We can set up a constraint (via SQL or store functions) so that if observationType is set to Biomass Measurements, it needs to be a adHoc monitoring plot.

I am looking at the requirements that require adhoc plots to be on the map outside the extent of planting zones or site.

Reply:
My guess is we'll end up having it in both places.

We definitely want it on monitoring plots so we can efficiently ignore existing ad-hoc plots when we're figuring out the plots to use for an assigned observation.

We may also want it on observations so we can easily filter the observations list based on ad-hocness.

As for plots being outside the site, I've already made the subzone ID nullable on monitoring plots and added a planting site ID to them. See #2660 .